packages/scenario/resources/logback-test.xml.example

```diff
@@ -0,0 +1,25 @@
+<configuration>
```

Comment:
🤷‍♂️

... I mean, no I don't think it is. In JavaRosa I see it referenced in `ResourcePathHelper.java`. The equivalent module here doesn't reference it.

The whole `resources` directory is basically copied as-is (apart from a few companion fixtures added after the fact to support alternate/supplemental tests). In hindsight:

- I kind of which I'd copied each fixture piecemeal, as each ported test required.
- I'd still want to copy any unused files, albeit maybe into a different place. Maybe we can also do a final pass to isolate any "resources"/fixtures that aren't currently referenced in any of the ported tests.

Reply:
I didn't want to forget to come back to this. Here's a list of resources that I don't see referenced in any test, or from any other fixture:

- bad-feature-not-feature.geojson
- bad-features-not-array.geojson
- bad-futures-collection.geojson
- calculate-now.xml
- constraint-message-error.xml
- external-secondary-comma-complex.csv
- external-secondary-csv-bom.csv
- external-secondary-instance-large.xml
- external-secondary-instance.xml
- external-secondary-semicolon-complex.csv
- external-select-xml-dummy-nodes.xml
- external-select-xml.xml
- feature-collection-extra-feature-toplevel.geojson
- feature-collection-extra-toplevel.geojson
- feature-collection-id-toplevel.geojson
- feature-collection-id-twice.geojson
- feature-collection-integer-id.geojson
- feature-collection-no-properties.geojson
- feature-collection-toplevel-order.geojson
- feature-collection-with-null.geojson
- feature-collection-with-unsupported-type.geojson
- feature-collection.geojson
- form_with_additional_attributes.xml
- form_with_bind_attributes.xml
- formindex-serialization.xml
- hhplotdetails.csv
- ImageSelectTester.xml
- invalid-type.geojson
- logback-test.xml.example
- nigeria_wards_external_combined.xml
- nigeria_wards_external.xml
- not-object.geojson
- populate-nodes-attributes-instance.xml
- populate-nodes-attributes.xml
- primary-instance-with-id.xml
- randomize.xml
- repeatGroupWithQuestionAndRegularGroupInside.xml
- repeatGroupWithTwoQuestions.xml
- simpleFormWithThreeQuestions.xml
- sms_form.xml
- twoNestedRegularGroups.xml
- twoNestedRepeatGroups.xml
- unused-secondary-instance.xml

Quite a few of these are in deeply nested subdirectories. I'm less inclined now to move them anywhere, just because it'll be fussy to do now, and might be fussy to track them back if they ever become pertinent in future work. But this list can be a point of reference of unused files for now.